.NET/Microsoft.Recognizers.Text.DateTime/Chinese/Parsers/ChineseMergedDateTimeParserConfiguration.cs

```diff
@@ -1,135 +0,0 @@
-﻿using System;
-using System.Text.RegularExpressions;
-using Microsoft.Recognizers.Definitions.Chinese;
-
-using DateObject = System.DateTime;
-
-namespace Microsoft.Recognizers.Text.DateTime.Chinese
```

Comment:
What happened to this class?

Reply:
Became ChineseMergedParserConfiguration.cs?

Reply:
yes, I changed the name to be in line with the other languages.